### DISCUSSION TITLE
Email to attendees not send

### DISCUSSION THREAD

**Discussion by chondaen12** [pretix/pretix#3199] [Q&A] [answered]
I am trying to sell a group ticket. I set up the product by creating a non-admission bundle with more admission products (the bundled product are set to only sold as part of bundle). The issue is when a customer creates an order and puts in other guest emails as an attendee, The email only sends to the customer who put in an order. Even though I have already enabled the "Send an email to attendees" option in the email content setting.  Is this the way it is intended to work, or am I missing something

**Comment by raphaelm** [comment, marked as answer]
Add-on products can not receive emails currently, sorry
